azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -139,9 +146,9 @@
   private boolean shouldProxy = false;
   private boolean securityEnabled = false;
 
-  private HadoopSecurityManager_H_2_0(final Props props)
+  private HadoopSecurityManager_H_2_0(Props props)
```

Comment:
Maybe I lead you down the wrong path, did you follow the guide to set up your local editor?

Generally we keep the `final` for all variables where it can be applied, as well as the `this.` prefaces.

We also tend not to preface the logger statements/other static variables with the class name, so the `logger.info` statements need not be updated nor should the `ucl` usage.

Reply:
I enabled the save actions plugin and turned on all the options (except the one mentioned in the guide) as the contribution guide says. These two options of the save actions plugin cause these changes:
 - Add class qualifier to static member access
 - Remove unnecessary final to local variable or parameter.

I'll change the code according to your comment, but i think the contribution guide needs to be updated.

Reply:
@aycaacar

Thanks

What update do you suggest?

Could you give examples of the issues you saw?

Reply:
Install and enable the intellij's save actions plugin to reformat/refactor code automatically:
Please turn on all the options except
 - Remove unused suppress warning annotation

I think this should be updated as follows:
Please turn on all the options except
 - Remove unused suppress warning annotation
 - Add this to method access
 - Add class qualifier to static method access
 - Remove unnecessary this
 - Remove unnecessary final to local variable or parameter

![image](https://user-images.githubusercontent.com/6359110/38080457-305244ae-334a-11e8-819c-eac31e4a45ea.png)

Reply:
Do you mean Java inspection options conflict with Save Action plugin options?

Where can I find these settings you referenced?

Reply:
The screenshot belongs to Save Actions plugin. 

The contribution guide tells us to check all the options (except one)in Save Actions plugin. But if we do it so, the code produced is not the way it is wanted. For example, it adds class qualifiers to static members which is not the desired behaviour. So I suggest that the contribution guide should be updated so that related options in Save Actions plugin are not required to be checked.

Reply:
Strange. I don't see the same screen.

Here is what I have

![screen shot 2018-03-29 at 10 52 20 am](https://user-images.githubusercontent.com/6646529/38105002-bb74786e-333f-11e8-8b01-c4ca4ceaf437.png)

I checked. I have the latest version of the plugin 0.16

Reply:
I have version 0.26 of the plugin.

Reply:
Got it. I haven't updated. Thanks for reporting the issue. I will discuss with the team and update the guide.